docker/package/tezos_baking_wizard.py

```diff
@@ -323,25 +324,10 @@ def get_client_data_directory(network):
     )
 
 
-def get_client_node_rpc_endpoint(network):
-    return search_baking_service_config(
-        network, 'NODE_RPC_ENDPOINT="(.*)"', "http://localhost:8732"
-    )
```

Comment:
But :why:? Granted, right now there's not much of a difference, because both ways would query `systemctl` once, `get_node_rpc_addr` for `tezos-node-<network>.service` and `get_client_node_rpc_endpoint` for `tezos-baking-<network>.service`. But if you do end up going through with saving the output from config file referenced in `EnvironmentFiles` (see my other remaining comment), then it's better to use the new way (and consequently remove the `get_node_rpc_addr` function completely) to 1) save an external process call 2) increase readability some

Reply:
Oh, looks like I completely misunderstood your previous comments. Sorry, my brain doesn't work very well today :(

Reply:
Please, check whether I got you correctly this time